modules/openxr/openxr_api_extension.cpp

```diff
@@ -193,6 +211,94 @@ void OpenXRAPIExtension::unregister_composition_layer_provider(OpenXRExtensionWr
 	OpenXRAPI::get_singleton()->unregister_composition_layer_provider(p_extension);
 }
 
+void OpenXRAPIExtension::register_projection_views_extension(OpenXRExtensionWrapperExtension *p_extension) {
+	ERR_FAIL_NULL(OpenXRAPI::get_singleton());
+	OpenXRAPI::get_singleton()->register_projection_views_extension(p_extension);
+}
+
+void OpenXRAPIExtension::unregister_projection_views_extension(OpenXRExtensionWrapperExtension *p_extension) {
+	ERR_FAIL_NULL(OpenXRAPI::get_singleton());
+	OpenXRAPI::get_singleton()->unregister_projection_views_extension(p_extension);
+}
+
+uint64_t OpenXRAPIExtension::get_projection_views() {
+	ERR_FAIL_NULL_V(OpenXRAPI::get_singleton(), 0);
+	return (uint64_t)OpenXRAPI::get_singleton()->get_projection_views();
+}
+
+uint64_t OpenXRAPIExtension::get_render_state_z_near() {
+	ERR_FAIL_NULL_V(OpenXRAPI::get_singleton(), 0);
+	double z_near = OpenXRAPI::get_singleton()->get_render_state_z_near();
+	return *reinterpret_cast<uint64_t *>(&z_near);
```

Comment:
This function should just return `double` - there's no reason to convert it to `uint64_t`